README.md

```diff
@@ -48,84 +59,126 @@ The script will generate solar forecasts at the specified intervals between the
 - Depending, whether the timestamp for the prediction lays more than 90 days in the past or not, different data sources for the NWP are used. If we predict within the last 90 days, we can use ICON or GFS from the open-meteo Weather Forecast API. Since ICON doesn't provide visibility, this parameter is queried from GFS in any case. If the date for the prediction is further back in time, a reanalysis model of historical data is used (open-meteo | Historical Weather API). The historical weather API doesn't't provide visibility at all, that's why it's set to a maximum of 24000 meter in this case. This can lead to some loss of precision.
+- The model was trained and tested only over the UK, applying it to other geographical regions should be done with caution.
+- When using the XGBoost model, only predictions within the last 90 days are available for data consistency.
 
 ## Evaluation
 
+**Gradient Boosting Model** (default)
+
 To evaluate the model we use the [UK PV](https://huggingface.co/datasets/openclimatefix/uk_pv) dataset and the [ICON NWP](https://huggingface.co/datasets/openclimatefix/dwd-icon-eu) dataset.
 All the data is publicly available and the evaluation script can be run with the following command
 
 ```bash
 python scripts/run_evaluation.py
 ```
 
-The test dataset we used is defined in `quartz_solar_forecast/dataset/testset.csv`. 
-This contains 50 PV sites, which 50 unique timestamps. The data is from 2021. 
-
-The results of the evaluation are as follows
-The MAE is 0.1906 kw across all horizons. 
+The test dataset we used is defined in `quartz_solar_forecast/dataset/testset.csv`.
+This contains 50 PV sites, which 50 unique timestamps. The data is from 2021.
 
 | Horizons | MAE [kw]      | MAE [%] |
-|----------|---------------| ------- |
-| 0        | 0.202 +- 0.03 | 6.2 |
-| 1        | 0.211 +- 0.03 | 6.4 |
-| 2        | 0.216 +- 0.03 | 6.5 |
-| 3 - 4    | 0.211 +- 0.02 |6.3 |
-| 5 - 8    | 0.191 +- 0.01 | 6 |
-| 9 - 16   | 0.161 +- 0.01 | 5 |
-| 17 - 24  | 0.173 +- 0.01 | 5.3 |
-| 24 - 48  | 0.201 +- 0.01 | 6.1 |
-
-If we exclude nighttime, then the average MAE [%] from 0 to 36 forecast hours is 13.0%. 
-
-Notes: 
-- The MAE in % is the MAE divided by the capacity of the PV site. We acknowledge there are a number of different ways to do this. 
+| -------- | ------------- | ------- |
+| 0        | 0.202 +- 0.03 | 6.2     |
+| 1        | 0.211 +- 0.03 | 6.4     |
+| 2        | 0.216 +- 0.03 | 6.5     |
+| 3 - 4    | 0.211 +- 0.02 | 6.3     |
+| 5 - 8    | 0.191 +- 0.01 | 6       |
+| 9 - 16   | 0.161 +- 0.01 | 5       |
+| 17 - 24  | 0.173 +- 0.01 | 5.3     |
+| 24 - 48  | 0.201 +- 0.01 | 6.1     |
```

Comment:
Could you add your models MAE in this table too?

Reply:
I thought you did in the presentation?

Reply:
In the presentation we explained that we cannot calculate errors with respect to the horizons using the open-meteo data. If you have more questions about this you can ask Eliana, she has been working on this and knows the context much better, than I do.

Reply:
ok, lets leave it for now - thanks